add tailscale as local ip range

diff --git a/src/NzbDrone.Common/Extensions/IpAddressExtensions.cs b/src/NzbDrone.Common/Extensions/IpAddressExtensions.cs
index 7feb431c4..6978b20a1 100644
--- a/src/NzbDrone.Common/Extensions/IpAddressExtensions.cs
+++ b/src/NzbDrone.Common/Extensions/IpAddressExtensions.cs
@@ -50,7 +50,10 @@ private static bool IsLocalIPv4(byte[] ipv4Bytes)
             // Class C private range: 192.168.0.0 – 192.168.255.255 (192.168.0.0/16)
             bool IsClassC() => ipv4Bytes[0] == 192 && ipv4Bytes[1] == 168;

-            return IsLinkLocal() || IsClassA() || IsClassC() || IsClassB();
+            // IANA Special range (RFC6598 also used by Tailscale): 100.64.0.0 – 100.127.255.255 (100.64.0.0/10)
+            bool IsSpecial() => ipv4Bytes[0] == 100 && ipv4Bytes[1] >= 64 && ipv4Bytes[1] <= 127;
+
+            return IsLinkLocal() || IsClassA() || IsClassC() || IsClassB() || IsSpecial();
         }
     }
 }
