add Repack revision for renaming instead of just Proper

--- a/src/NzbDrone.Core/Organizer/FileNameBuilder.cs
+++ b/src/NzbDrone.Core/Organizer/FileNameBuilder.cs
@@ -1097,6 +1097,11 @@ private string GetQualityProper(Series series, QualityModel quality)
                     return "v" + quality.Revision.Version;
                 }

+                if (quality.Revision.IsRepack)
+                {
+                    return string.Format("Repack{0}", quality.Revision.Version < 3 ? "" : quality.Revision.Version - 1);
+                }
+
                 return "Proper";
             }
